packages/slate-react/src/components/leaf.tsx

```diff
@@ -12,6 +12,7 @@ const Leaf = (props: {
   isLast: boolean
   leaf: Text
   parent: Element
+  renderPlaceholder: (props: RenderPlaceholderProps) => JSX.Element
```

Comment:
I think this new prop needs to be reflected in `MemoizedLeaf` below.

Reply:
good catch!